test/util/test_runners/integrations_tests_runner.py

```diff
@@ -0,0 +1,138 @@
+"""
+This is a runner module for all integration tests.
+The purpose of this file is to be a cross platform tests runner
+for all integration tests
+
+Roadmap:
+1) run all integration tests from this file
+2) set up the system env to run the tests
+"""
+
+# Builtins
+import sys
+import os
+
+# Third party
+import unittest
+import base64
+import requests
+from pathlib import Path
+import logging
+import subprocess as sb
+import shutil
+
+# Internals
+from test.test_integration_cli import CliIntegrationTest
+from conjur.endpoints import ConjurEndpoint
+from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
+from test.util.test_runners.test_runner_args import TestRunnerArgs
+from test.util.test_runners.params import ClientParams, TestEnvironmentParams
+from test.util.test_path_provider import TestRunnerPathProvider
+from conjur.constants import *
+
+
+def main():
+    args = TestRunnerArgs.create_from_args()
+
+    runner = TestRunner(args)
+    runner.start_env()
+    runner.run_tests(test_modules=[CliIntegrationTest])
```

Comment:
is this really running tests? or just gathering them
may be more representative if we say `accum_tests`